src/groups/group_action.jl

```diff
@@ -115,14 +115,15 @@ Compute the value of differential of action [`AbstractGroupAction`](@ref) `A` on
 where element `a` is acting on `p`, with respect to the group element.
 
 Let ``\mathcal G`` be the group acting on manifold ``\mathcal M`` by the action `A`.
-The action is of element ``g ∈ \mathcal G`` on a point ``p ∈ \mathcal M``.
-The differential transforms vector `X` from the tangent space at `a ∈ \mathcal G`, 
-``X ∈ T_a \mathcal G`` into a tangent space of the manifold ``\mathcal M``.
-When action on element `p` is written as ``\mathrm{d}τ^p``, with the specified left or right
-convention, the differential transforms vectors
+The action is of element ``a ∈ \mathcal G`` on a point ``p ∈ \mathcal M``,
+and is denoted ``τ^p(a)``.
```

Comment:
While `g` or now `a` was not listed in our notation at https://juliamanifolds.github.io/Manifolds.jl/latest/misc/notation.htmlm the group action is.
Do we have to change the notation? If so, we should do that throughout all of the package. Just fixing something a bit too locally already lead to #669, let's be more careful. Could we maybe continue to use ``τ_p``?

Reply:
The notation $τ^p$ was there before, I just reused it. Choose whatever notation is consistent throughout.

Reply:
Hm, I think I have to stop here – this was not my area when it was introduced and by now I am just totally lost.

Reply:
Ah we indeed used both, sorry, we should unify this to what is mentioned in the documentation